proposal.md

```diff
@@ -0,0 +1,36 @@
+# TEMPLATE
+
+## 🔥 Do not edit this file - copy the template and create your own file.
+
+### 📌 Step 1
+
+**TITLE:**
+Intro to Python Libraries: Build an Alexa Skill
+
+**TOPIC:**
+Intro to Python Libraries
+
+**DESCRIPTION (5-7+ sentences):**
+This workshop will have two main sections and focuses on introducing students to different libraries and the directions they could go with a versatile language like python. The workshop will overview core data science libraries like sci-kit learn, pandas, and may go into deep learning like tensorflow and keras to expose students. Web development in python with flask and django will be presented to show a different direction. Finally, I plan to express how you can use python to write scripts to bring your ideas to life. This final point will be implemented by walking the user through building their own roulette game alexa skill. 
+
+​	
+
+
+
+### 📌 Step 2
+
+👪 **TARGET AUDIENCE (3-5+ sentences):**
+The target audience of this workshop will be entry level programmers. They should have understandings of beginner level concepts like functions, lists etc but are looking to add more to their projects. I found that BitProject had many blogs and workshops in the works describing these libraries including an intoduction to pandas, flask and linear regression with scikit learn. This workshop's goal is to provide a description of pythons libraries to give beginners a launching point into one of the more technical blogs while also providing a fun project.
+
+### 📌 Step 3
+
+> Outline your learning/teaching structure:
+
+**Beginning (2-3+ sentences):**
+The beginning of the workshop will open with descriptions of python libraries. This will include descriptions of data science, web development and machine learning using python to give beginners direction.
+
+**Middle (2-3+ sentences):**
+The middle section will be used to show how Python scripts can be written outside of the major categories to fulfill your ideas. This will lead into walking the user through creating their own alexa skill. The skill will be a roulette style game and will show the user the versatility of this language. 
+
+**End (2-3+ sentences):**
```

Comment:
I liek the idea of letting the reader know what the intention is/the skills gained for the programmer.

Reply:
Please let me know if you have any suggestions going further.